lib/qualys/element.rb

```diff
@@ -81,5 +87,21 @@ def method_missing(method, *args)
         return nil
       end
     end
+
+    private
+      
+    def cleanup_html(source)
+      result = source.dup
+      result.gsub!(/<P>/, "\n\n")
```

Comment:
* Does Qualys only provide `<P>` tags and not the corresponding `</P>` ones?
* I think we should make all the regexes case insensitive just in case (e.g. `/<p>/i`) to cover both cases.